components/forms/farmer/Confirmation.tsx

```diff
@@ -0,0 +1,35 @@
+import { useFormContext } from "react-hook-form";
+import Label from "./Label";
+import PreviewField from "./PreviewField";
+import { FarmerJournal } from "@/types/content";
+
+export default function Confirmation() {
+  const {watch} = useFormContext<FarmerJournal>();
+  const data = watch();
```

Comment:
Not a blocker, but just curious:

Is this function working as expected? E.g. allowing us to display derived state?

Reply:
The function has nothing to do with the computed states. It's just for getting the value of all the fields.